src/promptflow/promptflow/_sdk/_service/app.py

```diff
@@ -94,4 +108,24 @@ def log_after_request_info(response):
             )
             return response
 
+        # Start a monitor process using detach mode. It will stop pfs service if no request to pfs service in 1h in
+        # python scenario. For C# scenario, pfs will live until the process is killed manually.
+        def monitor_request():
+            while True:
+                time.sleep(60)
```

Comment:
maybe log one line here, so we have a line of log every minute

Reply:
added